R/stao/commercial_crew_processing_114.R

```diff
@@ -0,0 +1,35 @@
+################
+#Issue #114: Commercial Crew Member Data
+#Data Processing
+#February 2018
+#Sophia Tao
+################
+
+
+
+a <- read.csv('/home/stao/my-sasap/114_commercial_crew/Commercial Crew data 2012-2016.csv', 
+              header = T, 
+              stringsAsFactors = F, 
+              na.strings = c("", "Not Available"))
```

Comment:
hmm it looks like `na.strings` solved the problem you had to deal with in the other file ("." --> NA)!  Cool, I didn't know about this argument!